src/main/java/TaskManager.java

```diff
@@ -0,0 +1,50 @@
+public class TaskManager {
+    private static int taskNo = 0;
+    final private static int maxTasks = 100;
+    final static Task[] tasks = new Task[maxTasks];
+
+    static void addTask(Task task) {
+        if (taskNo < maxTasks) {
```

Comment:
You can consider [making the happy path prominent](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#make-the-happy-path-prominent) here.